draft-packages/illustration/KaizenDraft/Illustration/Players/LottiePlayer.spec.tsx

```diff
@@ -100,4 +100,35 @@ describe("<AnimatedBase />", () => {
       expect(items).toHaveLength(1)
     })
   })
+
+  describe("Render", () => {
+    beforeEach(() => {
+      mockedGetAnimationData.getAnimationData.mockResolvedValue(
+        {} as LottieAnimation
+      )
+    })
+
+    it("should have aspect ratio class", () => {
+      const { container } = render(
+        <AnimatedBase
+          name=""
+          aspectRatio="landscape"
+          alt="Screen reader text"
+          fallback="illustrations/heart/spot/moods-cautionary.svg"
+        />
+      )
+      expect(container.querySelector(".landscape")).toBeTruthy()
+    })
+
+    it("should not have aspect ratio class", () => {
```

Comment:
👍 for adding these, but just reading these two tests side by side, I find it hard to grasp what scenarios they're testing (without reading the test setup in detail).

I would suggest either adding `".... when the aspect ratio is (not) set"` to the description of the test, or wrapping each in a `describe("When the aspect ratio is (not) set", ...)` block respectively.

Reply:
Yep good call with the descriptiveness. Unsure about the `(not)` though. Is that common language for a conditional in tests?

Reply:
Oh, what I mean by that is that there would be 2 `describes`:
```
describe("when the aspect ratio is set as a prop", ...)

describe("when the aspect ratio is not set as a prop", ...)
```
Is that clearer? In those circumstances I actually like to write it in capitals `...is NOT set...`, as that makes it quite obvious in the test runner printout.

Reply:
Cool that's what I thought, just wanted to check, ha.